runtime/src/androidMain/kotlin/pbandk/internal/Util.kt

```diff
@@ -0,0 +1,11 @@
+package pbandk.internal
+
+import android.util.Base64
+
+internal actual object Util : AbstractUtil() {
+    actual override fun base64ToBytes(str: String): ByteArray =
+        Base64.decode(str, Base64.DEFAULT)
+
+    actual override fun bytesToBase64(bytes: ByteArray): String =
+        Base64.encodeToString(bytes, Base64.DEFAULT)
```

Comment:
I think you forgot to update this based on the failing test

Reply:
Wow my, messed up the commits after wanting to split them up 🤦